Sources/MapboxCoreNavigation/RouteProgress.swift

```diff
@@ -191,14 +191,14 @@ open class RouteProgress: Codable {
         }
     }
 
-    func updateDistanceTraveled(with location: CLLocation, navigationStatus: NavigationStatus) {
+    func updateDistanceTraveled(navigationStatus: NavigationStatus) {
         let stepProgress = currentLegProgress.currentStepProgress
 
         if let activeGuidanceInfo = navigationStatus.activeGuidanceInfo {
             stepProgress.distanceTraveled = activeGuidanceInfo.stepProgress.distanceTraveled
         }
         else {
-            updateDistanceTraveled(with: location)
+            updateDistanceTraveled(with: .init(navigationStatus.location))
```

Comment:
But it changes nothing? `location` passed to `update(progress:status:with:rawLocation:upcomingRouteAlerts:mapMatchingResult:routeShapeIndex:)` is retrieved from `NavigationStatus` higher in the call hierarchy. This is refactoring, but not a fix for something.